openlibrary/macros/ReadingLogDropper.html

```diff
@@ -168,6 +168,9 @@ <h2>$_('Create a new list')</h2>
             </form>
         </div>
     </div>
+  $if work_key and edition_key and ctx.user.is_usergroup_member('/usergroup/beta-testers'):
+    $ last_read_date = get_latest_read_date(work_key, edition_key)
+    $:render_template('check_ins/check_in_prompt', work_key, users_work_read_status, edition_key=edition_key, last_read_date=last_read_date)
```

Comment:
This may not be the best place to render this component.  Maybe the list widget is better?

Reply:
We can refactor later, the challenge perhaps being if this gets reloaded on each "my books" card